lightning-invoice/src/utils.rs

```diff
@@ -128,6 +125,34 @@ fn _create_phantom_invoice<Signer: Sign, K: Deref>(
 		InvoiceDescription::Hash(hash) => InvoiceBuilder::new(network).description_hash(hash.0),
 	};
 
+	let keys = ExpandedKey::new(&keys_manager.get_inbound_payment_key_material());
```

Comment:
Maybe leave a comment here that if we ever see performance being too slow taking the `ExpandedKey` as a parameter is probably the first step.